cpu/em_tuned.py

```diff
@@ -0,0 +1,212 @@
+        """
+        Turn off tuned
+        """
+        status = process.system("tuned-adm off", shell=True)
+        self.log.info(" status %s " % status)
+        if status != 0:
+            self.error_count += 1
+            self.log.info("Error turning off tuned %s" % status)
+
+    def set_active_profile(self, profile):
+        """
+        To set active profile
+        """
+        status = process.system("tuned-adm profile %s" % profile, shell=True)
+        if status != 0:
+            self.error_count += 1
+            self.log.info("Error setting Profile %s" % profile)
+
+    def get_available_profiles(self):
+        """
+        To get all available profiles
+        """
+        available_profiles = []
+        cmd_profile = commands.getoutput('tuned-adm list').split('\n')
+        for line in cmd_profile:
+            if line.startswith("- "):
+                available_profiles.append(line.split("- ")[1].strip())
+        self.log.info("Profiles %s" % available_profiles)
+        return available_profiles
+
+    def get_active_profile(self):
+        """
+        To get the active profile
+        """
+        cmd_act_pf = "tuned-adm active|grep 'Current active profile:'|\
+                     awk '{print $4}'"
+        active_profile = process.system_output(cmd_act_pf, shell=True)
+        cmd_act_pf_lt = "tuned-adm list|grep 'Current active profile:'|\
+                         awk '{print $4}'"
+        act_profile_list = process.system_output(cmd_act_pf_lt, shell=True)
+        if not active_profile:
+            active_profile = None
+        if not act_profile_list:
+            act_profile_list = None
+        if active_profile != act_profile_list:
+            self.log.info("Active profile mismatch in tuned list"
+                          "and tuned active command")
+
+        return active_profile
+
+    def get_tuned_status(self):
+        """
+        Getting Tuned status
+        """
+        is_active = False
+        is_loaded = False
+        cmd_load = "systemctl status tuned.service|grep 'Loaded:'|\
+            awk '{print $2}'"
+        cmd_active = "systemctl status tuned.service|grep 'Active:'|\
+                      awk '{print $2}'"
+        output_load = process.system_output(cmd_load, shell=True)
+        output_active = process.system_output(cmd_active, shell=True)
+        self.log.info("Tuned status: %s" % output_active)
+        if output_load == 'loaded':
+            is_loaded = True
+        if output_active == 'active':
+            is_active = True
+        if output_active == 'failed':
+            is_active = False
+        if output_active == 'inactive':
+            is_active = False
+        return (is_loaded, is_active)
+
+    def tuned_start(self):
+        """
+        Starting Tuned service
+        """
+        self.log.info(": Starting tuned service")
+        cmd = "systemctl start tuned.service"
+        status = process.system(cmd)
+        if status != 0:
```

Comment:
can it not like 

if  process.system('systemctl start tuned.service' , ignore_status=True) != 0:

Reply:
I will make all the changes and update it. Closing this PR